Documentation/concepts/networking/routing.rst

```diff
@@ -0,0 +1,309 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+#######
+Routing
+#######
+
+.. _arch_overlay:
+
+Encapsulation
+=============
+
+When no configuration is provided, Cilium automatically runs in this mode as it
+is the mode with the least requirements on the underlying networking
+infrastructure.
+
+In this mode, all cluster nodes form a mesh of tunnels using the UDP based
+encapsulation protocols `VXLAN` or `Geneve`. All container-to-container network
+traffic is routed through these tunnels. 
+
+The overlay used by the mesh of tunnels is managed by the protocol of 
+your choice (`VXLAN` by default). It's up to the overlay implementation and
+the kernel's routing table to route traffic to the other nodes' overlays. 
+It's likely that the encapsulation protocol will pick up the default layer
+3 infrastructure for the host. 
```

Comment:
I'm not quite sure what this last sentence means. Is it trying to describe that it will likely follow the default route?